engine/src/test/java/io/zeebe/engine/processor/workflow/timer/TimerStartEventTest.java

```diff
@@ -56,14 +60,22 @@
 
   private static final BpmnModelInstance MULTI_TIMER_START_MODEL = createMultipleTimerStartModel();
 
-  private static final BpmnModelInstance FEEL_EXPRESSION_MODEL =
+  private static final BpmnModelInstance FEEL_DATE_TIME_EXPRESSION_MODEL =
       Bpmn.createExecutableProcess("process_5")
           .startEvent("start_5")
-          .timerWithCycleExpression("cycle(duration(\"PT1S\"))")
+          .timerWithDateExpression("date and time(date(\"3978-11-25\"),time(\"T00:00:00@UTC\"))")
```

Comment:
Scheduling a timer for the year 3978 sounds very optimistic :joy:

Reply:
Psst: duck/google it